lib/util/asset_token_ext.dart

```diff
@@ -44,6 +44,20 @@ extension AssetTokenExtension on AssetToken {
     return galleryThumbnailURL != null;
   }
 
+  String get secondaryMarketURL {
+    if (blockchain == "ethereum") {
```

Comment:
better use:
```
switch (blockchain) {
case "ethereum":...
case "tezos":...
default:
  return nil
}
```
We can skip opening new page for invalid url (eg: old bitmark token, or new blockchain in the future).